docs/security/trust/trust_sandbox.md

```diff
@@ -21,186 +21,124 @@ overview](content_trust.md).
 
+By using a docker-in-docker image for the `trustsandbox` container, you will also
+not pollute your real docker daemon cache with any images you push and pull.  The
+images will instead be stored in an anonymous volume attached to this container,
+and can be destroyed after you destroy the container.
 
 ## Build the sandbox
 
-In this section, you build the Docker components for your trust sandbox. If you
-work exclusively with the Docker Hub, you would not need with these components.
-They are built into the Docker Hub for you. For the sandbox, however, you must
-build your own entire, mock production environment and registry.
-
-### Configure /etc/hosts
-
-The sandbox' `notaryserver` and `sandboxregistry` run on your local server. The
-client inside the `notarysandbox` container connects to them over your network.
-So, you'll need an entry for both the servers in your local `/etc/hosts` file.
-
-1. Add an entry for the `notaryserver` to `/etc/hosts`.
-
-        $ sudo sh -c 'echo "127.0.0.1 notaryserver" >> /etc/hosts'
-
-2. Add an entry for the `sandboxregistry` to `/etc/hosts`.
-
-        $ sudo sh -c 'echo "127.0.0.1 sandboxregistry" >> /etc/hosts'
-
-
-### Build the notarytest image
-
-1. Create a `notarytest` directory on your system.
-
-        $ mkdir notarysandbox
-
-2. Change into your `notarysandbox` directory.
-
-        $ cd notarysandbox
-
-3. Create a `notarytest` directory then change into that.
-
-        $ mkdir notarytest
-        $ cd notarytest
-
-4. Create a filed called `Dockerfile` with your favorite editor.
-
-5. Add the following to the new file.
-
-        FROM debian:jessie
-
-        ADD https://master.dockerproject.org/linux/amd64/docker /usr/bin/docker
-        RUN chmod +x /usr/bin/docker \
-          && apt-get update \
-          && apt-get install -y \
-          tree \
-          vim \
-          git \
-          ca-certificates \
-          --no-install-recommends
-
-        WORKDIR /root
-        RUN git clone -b trust-sandbox https://github.com/docker/notary.git
-        RUN cp /root/notary/fixtures/root-ca.crt /usr/local/share/ca-certificates/root-ca.crt
-        RUN update-ca-certificates
-
-        ENTRYPOINT ["bash"]
-
-6. Save and close the file.
-
-7. Build the testing container.
-
-        $ docker build -t notarysandbox .
-        Sending build context to Docker daemon 2.048 kB
-        Step 1 : FROM debian:jessie
-         ...
-         Successfully built 5683f17e9d72
-
-
-### Build and start up the trust servers
-
-In this step, you get the source code for your notary and registry services.
-Then, you'll use Docker Compose to build and start them on your local system.
-
-1. Change to back to the root of your  `notarysandbox` directory.
+In this section, you'll use Docker Compose to specify how to set up and link together
+the `trustsandbox` container, the Notary server, and the Registry server.
+
+
+1. Create a filed called `trustsandbox.yml` with your favorite editor.  For example, using vim:
```

Comment:
I think we should create a regular `docker-compose.yml` file. That way they don't have to add the `-f trustsandbox.yml` when working with docker-compose. We should advice them to create a new directory first for the project ( e.g. `mkdir trustsandbox`)